suzieq/engines/pandas/address.py

```diff
@@ -96,22 +82,64 @@ def get(self, **kwargs) -> pd.DataFrame:
         if 6 in addr_types:
             df = df.explode('ip6AddressList').fillna({'ip6AddressList': ''})
 
+        v4addr = []
+        v6addr = []
         query_str = ''
-        prefix = ''
-        # IMPORTANT: Don't mess with this order of query. Some bug in pandas
-        # prevents it from working if macaddr isn't first and your query
-        # contains both a macaddr and an IP address.
-        if macaddr:
-            query_str += f'{prefix} macaddr.isin({macaddr}) '
-            prefix = 'or'
-        if v4addr:
-            for a in v4addr:
-                query_str += f'{prefix} ipAddressList.str.startswith("{a}") '
-                prefix = 'or'
-        if v6addr:
-            for a in v6addr:
-                query_str += f'{prefix} ip6AddressList.str.startswith("{a}") '
-                prefix = 'or'
+        filter_prefix = ''
+
+        # Address and prefix filtering are mutual exclusive
```

Comment:
I think it is better if you do a test like 
`if len(addr) > 0 and len(prefix) > 0:
  raise error "Cannot use both commands"`